test/jest/Corporation.test.ts

```diff
@@ -112,4 +116,59 @@ describe("Corporation", () => {
       expectSharesToAddUp(corporation);
     });
   });
+
+  describe("helpers.calculateOfficeSizeUpgradeCost", () => {
+    // These values were pulled from v2.6.0 as reference test values
+    const refCosts = {
+      from3: {
```

Comment:
Here's the results of a `bc` program (bc is the unix-standard arbitrary-precision calculator):
```bc
define cost(size, increase) {
  auto i, mult, old_scale, incr3, size3;
  old_scale = scale;
  scale = 0;
  incr3 = increase / 3;
  size3 = size / 3;
  scale = old_scale;
  for (; i < incr3; ++i) {
    mult += 1.09 ^ (size3 + i);
  }
  return 4000000000 * mult;
}

scale=40
cost(3, 3)
cost(3, 15)
cost(3, 150)
print "\n"
cost(6, 3)
cost(6, 15)
cost(6, 150)
print "\n"
cost(9, 3)
cost(9, 15)
cost(9, 150)
quit
```
```
4360000000.00
26093338259.6000000000
3553764305895.2490263450780533572274291097124000000000

4752400000.0000
28441738702.964000000000
3873603093425.8214387161350781593778977295868000000000

5180116000.000000
31001495186.23076000000000
4222227371834.1453682005872351937219085252500000000000
```
As you can see, there was some mild inaccuracy in the old method. I'd use these numbers (rounded to the same number of digits as you have here) instead.

Reply:
It's probably been decades since I last used `bc` and I never knew you could define functions in it.  That's handy to know about, thanks!

Reply:
I ended up just removing the least significant digit of each number until eslint stopped complaining that I'd lost precision at runtime.